docs/project/2024/amalia.munteanu/index.md

```diff
@@ -0,0 +1,88 @@
+# The Hangmaster
+A hangman game programmed on a Raspberry Pi Pico. 
+
+
+**Author**: Munteanu Amalia-Nicole \
+**GitHub Project Link**: link_to_github
+
```

Comment:
```suggestion
:::info

**Author**: Munteanu Amalia-Nicole \
**GitHub Project Link**: link_to_github

:::
```

Please add a link to your project's repository.

Reply:
okay